src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -62,6 +63,28 @@ internal static IntPtr LoadNativeLibrary()
             private static void PlatformInitialize()
             {
                 LibraryResolver.EnsureRegistered();
+
+                // Check whether the version of libgdiplus is a recent version of libgdiplus.
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
As discussed, we're not going to introduce this exception in the product as since we don't distribute libgdiplus with use, we can't enforce users to run on a specific version, but we can only recommend it in our docs.

Let's make this a test just to ensure our test machines have the right version